README.md

```diff
@@ -54,7 +53,12 @@
 
 <p align="right">(<a href="#readme-top">back to top</a>)</p>
 
+<!-- LIVE DEMO -->
+
+## 🚀 Live Demo <a name="live-demo"></a>
 
+- Link to live project: [Capstone project - A website for an upcoming conference](https://debygrey.github.io/web-development-course-website/)
+- Link to project demo: [Capstone project - Project demo](https://www.loom.com/share/b736aa959e014557a41959c866004862)
```

Comment:
- [ ] Great job recording the presentation video and including it in the readme file. :+1: , But I think there is a requirement you missed here. According to the project requirement, the MAXIMUM LENGTH of the video must be 5 MINS. But I noticed your video is 5:30 mins in length, As following all the requirements is part of what's needed to pass this milestone, I'll suggest you edit the video or quickly make another video within the required length. :+1: :100: 
> Once you have finished the development of the project, you should record a video presenting the features of the project you built. It is a video with a maximum length of 5 minutes. The content of the video should include: